tasks/4-deploy.yaml

```
@@ -111,6 +111,12 @@ spec:
           export CHART_ROOT=$(find . -name chart)
           echo "CHART_ROOT: $CHART_ROOT"
 
+          FILE_NUM=$(find . -name Chart*.yaml |wc -l)
+          if [[ "${FILE_NUM}" -gt 1 ]]; then
+            echo "Error: Found >1 Chart*.yaml"
+            exit 1
+          fi
+
           export CHART=$(find . -name Chart*.yaml)
```

```suggestion
          export CHART=$(find . -name 'Chart*.yaml')
```
probably here you need to quote it also

---

Done. Thanks.